inlong-common/src/main/java/org/apache/inlong/common/util/NetworkUtils.java

```diff
@@ -17,59 +17,55 @@
 
 package org.apache.inlong.common.util;
 
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.net.InetAddress;
 import java.net.NetworkInterface;
 import java.net.SocketException;
 import java.util.Enumeration;
 
 public class NetworkUtils {
-    public static String INNER_NETWORK_INTERFACE = "eth1";
 
-    private static String localIp = null;
+    private static final Logger LOGGER = LoggerFactory.getLogger(NetworkUtils.class);
+    private static String localIp;
 
     static {
         localIp = getLocalIp();
     }
 
     /**
      * get local ip
-     * @return
      */
     public static String getLocalIp() {
-        if (localIp == null) {
-            String ip = null;
-            Enumeration<NetworkInterface> allInterface;
-            try {
-                allInterface = NetworkInterface.getNetworkInterfaces();
-                for (; allInterface.hasMoreElements(); ) {
-                    NetworkInterface oneInterface = allInterface.nextElement();
-                    String interfaceName = oneInterface.getName();
-                    if (oneInterface.isLoopback()
-                            || !oneInterface.isUp()
-                            || !interfaceName
-                            .equalsIgnoreCase(INNER_NETWORK_INTERFACE)) {
-                        continue;
-                    }
+        if (!StringUtils.isEmpty(localIp)) {
+            return localIp;
+        }
+        Enumeration<NetworkInterface> allInterface;
+        String ip = "127.0.0.1";
+        try {
+            allInterface = NetworkInterface.getNetworkInterfaces();
+            for (; allInterface.hasMoreElements(); ) {
```

Comment:
Suggest changing the `for` to `while` or `forEach`.

Reply:
> Suggest changing the `for` to `while` or `forEach`.

done